docs/source/user_guide/cuspatial_api_examples.ipynb

```diff
@@ -133,7 +133,8 @@
     "import cudf\n",
```

Comment:
<p>Ok, I see why these values changed. Perhaps we should set a random seed in the notebook to prevent this from happening as much?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>